### PR TITLE
feat/updated examples with server work dir arg

### PR DESCRIPTION
**Pull request by vnamdeo**
@franknli Could you please review these changes and approve it accordingly. Thanks!

These examples have following changes:

1. Specified "release" and "server_workdir" arguments to showcase to the user (though these options are default)
2. Specified "upload_file_from_example_repo" API to download and upload input files from example repo to server working directory. With the usage of this API, we can track the issues if it's occurring with the future code changes.

### CODE REVIEW

**Comment by franknli** [comment]
> @franknli Could you please review these changes and approve it accordingly. Thanks!
> 
> These examples have following changes:
> 
> 1. Specified "release" and "server_workdir" arguments to showcase to the user (though these options are default)
> 2. Specified "upload_file_from_example_repo" API to download and upload input files from example repo to server working directory. With the usage of this API, we can track the issues if it's occurring with the future code changes.

@vnamdeo Using "upload_file_from_example_repo" is definitely a nice improvement! However, I am not sure about specifying release and server_workdir, per our private chat.

**Comment by vnamdeo** [comment]
> > @franknli Could you please review these changes and approve it accordingly. Thanks!
> > These examples have following changes:
> > 
> > 1. Specified "release" and "server_workdir" arguments to showcase to the user (though these options are default)
> > 2. Specified "upload_file_from_example_repo" API to download and upload input files from example repo to server working directory. With the usage of this API, we can track the issues if it's occurring with the future code changes.
> 
> @vnamdeo Using "upload_file_from_example_repo" is definitely a nice improvement! However, I am not sure about specifying release and server_workdir, per our private chat.

@franknli I have modified these examples as per our discussion. Please take a look for the further approval. Thanks!

**Comment by franknli** [comment]
> > @franknli Could you please review these changes and approve it accordingly. Thanks!
> > These examples have following changes:
> > 
> > 1. Specified "release" and "server_workdir" arguments to showcase to the user (though these options are default)
> > 2. Specified "upload_file_from_example_repo" API to download and upload input files from example repo to server working directory. With the usage of this API, we can track the issues if it's occurring with the future code changes.
> 
> @vnamdeo Using "upload_file_from_example_repo" is definitely a nice improvement! However, I am not sure about specifying release and server_workdir, per our private chat.

Sure. See my review comments!

Also, since we are now uploading these data files from example_repo, can we remove them from this repo - all the wbpz/agdb/cdb subdirectories can now be removed?

**Comment by vnamdeo** [comment]
> > > @franknli Could you please review these changes and approve it accordingly. Thanks!
> > > These examples have following changes:
> > > 
> > > 1. Specified "release" and "server_workdir" arguments to showcase to the user (though these options are default)
> > > 2. Specified "upload_file_from_example_repo" API to download and upload input files from example repo to server working directory. With the usage of this API, we can track the issues if it's occurring with the future code changes.
> > 
> > 
> > @vnamdeo Using "upload_file_from_example_repo" is definitely a nice improvement! However, I am not sure about specifying release and server_workdir, per our private chat.
> 
> Sure. See my review comments!
> 
> Also, since we are now uploading these data files from example_repo, can we remove them from this repo - all the wbpz/agdb/cdb subdirectories can now be removed?

@franknli Thanks for the review. I have removed subdirectories which are not required. Thanks!

**Review by abhishekchitwar — APPROVED** [review]

**Review by franknli — CHANGES_REQUESTED** [review]

File: examples/cooled-turbine-blade/main.py [hunk 1]
```diff
@@ -19,16 +19,19 @@
...
 wb = launch_workbench(client_workdir=str(workdir.absolute()))
 
-# Upload project files to the server using the `upload_file` method. 
+# Upload the project files to the server using the `upload_file_from_example_repo` method. 
 # The file to upload is `cooled_turbine_blade.wbpz`.
 
-wb.upload_file(str(wbpz / "cooled_turbine_blade.wbpz"))
+
+wb.upload_file_from_example_repo("cooled-turbine-blade/wbpz/cooled_turbine_blade.wbpz")
+wb.upload_file(str(scripts / "cooled_turbine_blade.py"))
```
[the remark this answers is not in the document]
Comment (vnamdeo) [reply examples/cooled-turbine-blade/main.py]: Done @franknli

File: examples/cyclic-symmetry-analysis/main.py [hunk 2]
```diff
@@ -16,16 +16,18 @@
...
 
 wb = launch_workbench(client_workdir=str(workdir.absolute()))
 
-# Upload project files to the server using the `upload_file` method. 
+# Upload the project files to the server using the `upload_file_from_example_repo` method. 
 # The file to upload is `sector_model.cdb`.
 
-wb.upload_file(str(cdb / "sector_model.cdb"))
+wb.upload_file_from_example_repo("cyclic-symmetry-analysis/cdb/sector_model.cdb")
+wb.upload_file(str(scripts / "cyclic_symmetry_analysis.py"))
```
[the remark this answers is not in the document]
Comment (vnamdeo) [reply examples/cyclic-symmetry-analysis/main.py]: Done @franknli

**Review by franknli — APPROVED** [review]
Looks great! Thanks.

**Pull request merged**